preprocess.py

```diff
@@ -165,6 +165,12 @@
         prop_spec["x-omitempty"] = False
         logging.info(f"set x-omitempty = false on WritableIPAddress.{prop}")
 
+# Remove omitempty = false for token write_enabled
```

Comment:
Sorry to nitpick, but this ADDS `omitempty = false` ;)

Reply:
No, thank you for nitpicking!